village/src/App.js

```diff
@@ -3,6 +3,10 @@ import React, { Component } from 'react';
 import './App.css';
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
+import axios from 'axios';
+import { BrowserRouter as Route, NavLink } from 'react-router-dom';
```

Comment:
You'll want to import BrowserRouter as Router. Route is already a defined object in react-router-dom. It could get confusing if you import BrowserRouter as something that's already defined.

Reply:
So after seeing you import BrowserRouter on your index file, which is correct, I realized what's going on here. When you want the Route component, you can bring it in like this import { Route, NavLink } from "react-router-dom";

Reply:
BrowserRouter is actually a different component from the react-router-dom package.